personalized_nlp/experiments/goemo_avg_ann_notune.py

```diff
@@ -0,0 +1,100 @@
+import os
+from itertools import product
+import torch
+
+from personalized_nlp.learning.train import train_test
+from personalized_nlp.models import models as models_dict
+from personalized_nlp.settings import LOGS_DIR
+from personalized_nlp.datasets.goemotions.go_emotions import GoEmotionsDataModule
+from personalized_nlp.utils import seed_everything
+from pytorch_lightning import loggers as pl_loggers
+
+torch.cuda.empty_cache()
+os.environ["CUDA_VISIBLE_DEVICES"] = "0"
+os.environ["WANDB_START_METHOD"] = "thread"
+
+if __name__ == "__main__":
+    wandb_project_name = 'goemo_baseline_avg_ann_notune'
+    regression = False
+    datamodule_cls = GoEmotionsDataModule
+    embedding_types = ['roberta']
+    model_types = ['baseline']
+    fold_nums = 10
+
+    min_word_counts = [50]
+    words_per_texts = [15]
+    
+    batch_size = 16
+    dp_embs = [0.25]
+    embedding_dims = [50]
+    epochs = 20
+    lr_rate = 3e-5
+
```

Comment:
I think, that for experiments with no fine-tuning you should declare `frozen` and `frozen_epochs_nr` parameters to be sure that the correct one are used. Now be default they are resepctively `False` and `5` so it is not experiment with no finetuning.